cydarm.json

```diff
@@ -0,0 +1,1988 @@
+        },
+        {
+            "action": "create case playbook",
+            "identifier": "create_case_playbook",
+            "description": "Add a playbook to a case",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "case_uuid": {
+                    "name": "case_uuid",
+                    "description": "Case UUID",
+                    "order": 0,
+                    "data_type": "string",
+                    "required": true
+                },
+                "playbook_uuid": {
+                    "name": "playbook_uuid",
+                    "description": "Playbook UUID",
+                    "order": 1,
+                    "data_type": "string",
+                    "required": true
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.case_uuid",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.playbook_uuid",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.acl",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.uuid",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get case playbook",
+            "identifier": "get_case_playbook",
+            "description": "Get a playbook instance associated with a case",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {
+                "case_uuid": {
+                    "name": "case_uuid",
+                    "description": "Case UUID",
+                    "order": 0,
+                    "data_type": "string",
+                    "required": true
+                },
+                "case_playbook_uuid": {
```

Comment:
Would be good to add a `contains`

Reply:
I had a read of this: https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/Contains#:~:text=The%20contains%20types%2C%20in%20conjunction,one%20of%20its%20output%20items.
Is there a CEF type for GUIDs? Or what would you recommend?

Reply:
You could do something like `cydarm playbook uuid`. Similar can go for other objects, such as `cydarm case uuid`. Basically, you would want to categorize the values so that they can be auto-populated fields when running other actions.

Reply:
Added contains field for UUID related params